File: hw2.py
import sys

def hw2():
	print('\nInterpreter\n'
			'Author: Sean Frischmann\n'
			'Class: Cse 305\n')
	while True:
		buf = raw_input('repl> ')
		str(buf)
		user_input = ''
		i=0
		while i < len(buf):
			if (buf[i] == ' '):
				if( user_input == 'quit'):
						sys.exit(0)
			user_input = user_input + buf[i]
			i=i+1

File: hw2.ml
(*===========================================================================
 * +++hw2.ml+++|
 * ____________|
 *
 *Author: Sean Frischmann
 *Class: Cse 305
 *=========================================================================*)

fun print_stack([]) = ()
   | print_stack(stack) =
       let
         val temp = stack;
         val head = hd(temp);
         val tail = tl(temp);
       in
         print(head);
         print("\n");
         print_stack(tail);
      end;

fun hw2 =
    let
         val stack = [];
         val splitter = String.tokens (fn c => c = #" ");
         val buf = "";
         val input = "";
    in
         print("repl> ");
         buf = TextIO.inputLine TextIO.stdIn;
         input = splitter(buf);
         print(buf);
         print(input);
   end;
